cloudflare/data_source_load_balancer_pools_test.go

```diff
@@ -0,0 +1,34 @@
+package cloudflare
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccCloudflareLoadBalancerPools(t *testing.T) {
+	t.Parallel()
+
+	rnd := generateRandomResourceName()
+	name := fmt.Sprintf("data.cloudflare_load_balancer_pools.%s", rnd)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCloudflareLoadBalancerPoolsConfig(rnd),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(name, "pools.#", "2"),
```

Comment:
The value here was tested against my own account data, I could not find any information about what to use for upstream acceptance tests.

Reply:
you can create them all as a part of your test case and assert against that state. `data_source_zones_test` has a good example https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/cloudflare/data_source_zones_test.go#L228-L267